evap/staff/tests/test_views.py

```diff
@@ -84,9 +84,40 @@ def test_deletes_users(self):
         self.assertEqual(UserProfile.objects.count(), user_count_before - 2)
 
 
+class TestSemesterView(ViewTest):
+    url = '/staff/semester/1'
+    test_users = ['staff']
+    extra_environ = {'HTTP_ACCEPT_LANGUAGE': 'de'}
+
+    @classmethod
+    def setUpTestData(cls):
+        mommy.make(UserProfile, username='staff', groups=[Group.objects.get(name='Staff')])
+        cls.semester = mommy.make(Semester)
+        cls.course1 = mommy.make(Course, name_de="A - Course 1", name_en="B - Course 1", semester=cls.semester)
+        cls.course2 = mommy.make(Course, name_de="B - Course 2", name_en="A - Course 2", semester=cls.semester)
+        mommy.make(Contribution, course=cls.course1, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+        mommy.make(Contribution, course=cls.course2, responsible=True, can_edit=True, comment_visibility=Contribution.ALL_COMMENTS)
+
+    def test_view_list_sorting(self):
+        self.extra_environ['HTTP_ACCEPT_LANGUAGE'] = 'en'
+
+        page = self.app.get(self.url, user='staff').body.decode("utf-8")
+        position_course1 = page.find("Course 1")
+        position_course2 = page.find("Course 2")
+        self.assertGreater(position_course1, position_course2)
+
+        self.extra_environ['HTTP_ACCEPT_LANGUAGE'] = 'de'
+
+        page = self.app.get(self.url, user='staff').body.decode("utf-8")
+        position_course1 = page.find("Course 1")
+        position_course2 = page.find("Course 2")
+        self.assertLess(position_course1, position_course2)
+
+
 class TestSemesterExportView(ViewTest):
     url = '/staff/semester/1/export'
     test_users = ['staff']
+    extra_environ = {'HTTP_ACCEPT_LANGUAGE': 'en'}
```

Comment:
is this still necessary here (other test as well)? if not, i'd prefer to just set this inside the test